fhir-swagger-generator/src/main/java/com/ibm/fhir/openapi/generator/FHIROpenApiGenerator.java

```diff
@@ -1247,7 +1249,7 @@ private static void generateProperty(StructureDefinition structureDefinition, Cl
             property.add("format", "int32");
         } else if (com.ibm.fhir.model.type.Base64Binary.class.equals(fieldClass)) {
             property.add("type", "string");
-            property.add("pattern","(\\s*([0-9a-zA-Z\\+\\=]){4}\\s*)+");
```

Comment:
I think the regex we use here matches the one given in the spec at https://www.hl7.org/fhir/datatypes.html#base64Binary

I do think `/` is a legal character though, so we should add that.  Are you sue that `=` doesn't need to be escaped inside a regex "character class" (square brackets)?

Reply:
ok, I double-checked our regex and it looks fine.  the corresponding issue to get the regex fixed in the spec is https://jira.hl7.org/browse/FHIR-28139